src/Clay/Media.hs

```diff
@@ -132,3 +150,21 @@ dpi i = Resolution (value (pack (show i) <> "dpi"))
 dppx :: Integer -> Resolution
 dppx i = Resolution (value (pack (show i) <> "dppx"))
 
+-------------------------------------------------------------------------------
```

Comment:
Rather use a haddock separator. But also here I'm unsure how this mixes with haddocks in the export list.
```suggestion
-- * Color scheme
```

Reply:
Ok, I noticed now that the rest of the file uses your style of separator. Fine as well.